src/dali/plugin/pair_converter/csv/kraken.py

```diff
@@ -62,8 +62,8 @@
 _GOOGLE_DRIVE_DOWNLOAD_URL: str = "https://drive.usercontent.google.com/download"
 
 # File ID for the unified CSV file ID. This will need to be replaced every quarter.
-# File can also be manually downloaded from https://drive.google.com/file/d/11WtjXA9kvVYV9KDoebGV5U75dmcA3bJa/view?usp=sharing
-_UNIFIED_CSV_FILE_ID: str = "11WtjXA9kvVYV9KDoebGV5U75dmcA3bJa"
+# File can also be manually downloaded from https://drive.google.com/file/d/1ptNqWYidLkhb2VAKuLCxmp2OXEfGO-AP/view?usp=sharing
```

Comment:
I'm trying to download this file to scan it with antivirus software, but it gets throttled around 75% and never finishes the download. I'll try again tomorrow.

Reply:
There is actually a comment in the header that already explains that and has a link to the article. I updated the download link in that comment though.